GOVERNANCE.md

```diff
@@ -1,172 +1,72 @@
-  1. An unmovable deadline is threatened by continuing the Consensus process; or
-  2. A Decider feels there is unreasonable blocking of both reaching Consensus and agreeing to a vote.
-      This should be rare, due to the social cost of discontinuing the Consensus process for this reason.
-      Most decisions should wait for the above process to take its course.
-- If Deciders agree to a vote, the default is a Simple Majority.
-- However, there are cases that require stronger voting – Supermajority or Unanimity – specified below:
-
-### Simple Majority Decisions
+- Engage deeply in the work being done by the working group - that is, the role is not purely administrative.
+  Chairs are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
+- When necessary, handle human and technical coordination across different working group workstreams
 
-If a vote is called, the default is a Simple Majority Vote <https://en.wikipedia.org/wiki/Majority>.
+### Current Chairs
 
-### Supermajority Decisions
+Name | Organization | GitHub
+-- | -- | --
+Dan Garfield | Codefresh | @todaywasawesome
+Leonardo Murillo | Cloud Native Architects | @murillodigital
+Scott Rigby | Weaveworks | @scottrigby
 
-If a vote is called, the following decisions require a Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority>.
+## Elections
 
-- Oversight Committee: Enforcing a Code of Conduct violation by a community member.
-- Oversight Committee: Licensing and intellectual property changes.
-- Oversight Committee: Material changes to the Governance document.
-  - Note: editorial changes to governance may be made by lazy consensus, unless challenged.
-    These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
-    They do not change the intention or meaning of anything in this document.
+[Chairs](#chairs) will be elected for one year terms and may run for reelection.
 
-### Unanimity Decisions
+Chairs are selected by the members of the working group.
 
-If a vote is called, the following decisison require Unanimity <https://en.wikipedia.org/wiki/Unanimity>.
+The election process takes place on the CNCF SIG App Delivery [mailing list](cncf-sig-app-delivery@lists.cncf.io ).
 
-- Repository Maintainers: Electing new Maintainers of the same repository.
-- Oversight Committee: Electing new Committee members.
-- Oversight Committee: Removing a Repository Maintainer or Committee member for any reason other than inactivity.
+Before each election, there will be a call for nominiations on the mailing list.
+Nominations should be sent to the mailing list, and include a brief bio and personal statement describing the candidates qualifications to serve in this capacity.
+Self-nominations are welcome.
 
-## Proposal Process
+Any member may cast a vote.
```

Comment:
This was my initial super-simple stab – assuming good will all around – though as written now it would be pretty easy to game the system if membership is self-declared and everyone can vote. We may want to do something similar to what CNF WG does, by having [interested parties](https://github.com/cncf/cnf-wg/blob/master/GOVERNANCE.md#elections) register within a certain time period before elections. I think our voting rules can be simpler though, since we aren't trying to balance 3 categorically different communities as they are. Thougths?

Reply:
I added an interested parties doc, with people and orgs listed in the initial GitOps WG charter.